reactor-core/src/main/java/reactor/core/publisher/Hooks.java

```diff
@@ -423,7 +424,7 @@ public static void resetOnNextDropped() {
 	}
 
 	@Nullable
-	static BiFunction<? super Throwable, Object, ? extends Throwable> createOrUpdateOpErrorHook(Collection<BiFunction<? super Throwable, Object, ? extends Throwable>> hooks) {
+	static BiFunction<? super @NonNull Throwable, @Nullable Object, ? extends @NonNull Throwable> createOrUpdateOpErrorHook(Collection<@NonNull BiFunction<? super @NonNull Throwable, @NonNull Object, ? extends @NonNull Throwable>> hooks) {
```

Comment:
the right-hand side `Object` in the parameter `BiFunction` is also `@Nullable` ;)